sdk/storage/storage-queue/src/policies/RetryPolicy.ts

```diff
@@ -59,7 +59,7 @@ const DEFAULT_RETRY_OPTIONS: RetryOptions = {
   tryTimeoutInMs: 30 * 1000 // https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-queue-service-operations
 };
 
-const RETRY_ABORT_ERROR = new RestError("The request was aborted", RestError.REQUEST_ABORTED_ERROR);
+const RETRY_ABORT_ERROR = new RestError("The operation was aborted.", RestError.REQUEST_ABORTED_ERROR);
```

Comment:
Shouldn't this be `AbortError`?

Reply:
Yes, missed a couple spots but after removing `RestError.REQUEST_ABORTED_ERROR` believe I've updated them all now.